geode-core/src/integrationTest/java/org/apache/geode/cache/query/internal/DefaultQuerySecurityIntegrationTest.java

```diff
@@ -76,7 +78,7 @@ private void createAndPopulateRegion(String name, RegionShortcut shortcut, int e
   }
 
   @Before
-  public void setUp() {
+  public void setUp() throws ServiceNotFoundException {
```

Comment:
> Random question: what happens if multiple servers have different authorizers configured within the `cache.xml` file when they start up?. Should we add some safe guard to prevent this, as we do to avoid multiple members having the same region with different attributes?, something similar?, nothing at all?.

If I recall correctly, there are other elements of `cache.xml` that also do not check to confirm that all servers have the same settings, so while it's possible that a user could create a situation where different servers have different authorizers configured, it's not something I think we need to check here. Using cache.xml at all is generally discouraged at this point, and the main expected use case will be configuration through gfsh and cluster config, so this should not present a major risk.

Reply:
Fair enough, I added a [comment](https://issues.apache.org/jira/browse/GEODE-6994?focusedCommentId=16961921&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16961921) within the documentation ticket to make sure this gets captured.